component/navbar.js

```diff
@@ -0,0 +1,37 @@
+import styles from "../styles/Navbar.module.css";
+export default function Navbar({children}){
```

Comment:
can we rename this to a more functional style. Something like `export const Navbar: NextPage = ({children}) => {}` Take a look at the home page for a reference.